spec/models/validation_report_spec.rb

```diff
@@ -26,24 +34,53 @@
         .to raise_error RSolr::Error::Http
     end
 
-    it 'gets by provider_id' do
-      provider_uri = RDF::URI(Krikri::Settings.prov.provider_base) / '123'
-      params = { :qt => 'standard',
-                 :q => '-sourceResource_title:[* TO *]',
-                 :rows => '10',
-                 :fq => "provider_id:\"#{provider_uri}\"" }
+    context 'with missing values in record' do
+      include_context 'with missing values'
 
+      it 'gets docs for all providers' do
+        docs = subject.find('sourceResource_title').response['docs']
 
-      response = double('solr response')
-      allow(Krikri::SolrResponseBuilder).to receive(:new).with(params)
-                                             .and_return(response)
-      allow(response).to receive(:response).and_return(:result)
+        expect(docs.map { |d| d['id'] })
+          .to contain_exactly(empty.rdf_subject.to_s,
+                              empty_new_provider.rdf_subject.to_s)
+      end
 
-      result = subject.find('sourceResource_title') do
-        self.provider_id = '123'
+      it 'allows limiting of rows per page' do
+        subject.rows = 1
+        expect(subject.find('sourceResource_title').docs.count).to eq 1
       end
 
-      expect(result).to eq :result
+      it 'allows pagination' do
+        subject.rows = '1'
+        page_one_docs = subject.find('sourceResource_title').docs
+        subject.page = '2'
+        expect(subject.find('sourceResource_title').docs)
+          .not_to eq page_one_docs
+      end
+
+      it 'multilies default pagination' do
```

Comment:
(minor) typo: maybe something like `it 'paginates based on a default number of rows'`